tests/resources/rospkg3/include/rospkg/.hpp

```diff
@@ -0,0 +1,3 @@
+#ifndef ROSPKG__FOOBAR_HPP_
```

Comment:
What's this file? :thinking:

Reply:
Oops. It's a mistake. https://github.com/tier4/pre-commit-hooks-ros/pull/32/commits/18ed048775c5e43282fad3ad02d0ec7b27b40e96